### DISCUSSION TITLE
Minecraft crash

### DISCUSSION THREAD

**Discussion by Skgaming1254** [FabricMC/fabric-api#3460] [FAQ]
My Minecraft being crashed after I put bunch of mods in it buts are fully compatible I put every dependency mod that they needed.
Here is crash log.
[crash-2023-12-10_16.08.36-client.txt](https://github.com/FabricMC/fabric/files/13627435/crash-2023-12-10_16.08.36-client.txt)

**Comment by Skgaming1254** [comment]
Your Minecraft found a wife and he don't want to be played anymore he loves his wife and he is doing very ❤️‍🩹 😭

**Comment by Poopooracoocoo** [comment]
Wrong version of Ukulib
